docs/content/preview/manage/backup-restore/instant-db-cloning.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Instant database cloning
+headerTitle: Instant database cloning
+linkTitle: Instant database cloning
+description: Clone your database in YugabyteDB for data recovery, development, and testing.
+badges: tp
+menu:
+  preview:
+    identifier: instant-db-clone
+    parent: backup-restore
+    weight: 706
+type: docs
+---
+
+Instant database cloning in YugabyteDB allows you to quickly create a zero-copy, independent writable clone of your database that can be used for data recovery, development, and testing. Cloning is both fast and efficient because when initially created, it shares the same data files with the original database. Subsequently, as data is written to the clone, the clone stores its own changes as separate and independent delta files. Although they physically share some files, the two databases are logically isolated, which means you can freely play with the clone database, perform DDLs, read and write data, and delete it without affecting the original database.
+
+You can create clones as of now, or as of any time in the recent past, within a configurable history retention period. This is particularly useful for data recovery due to user or application error scenarios. For instance, if you dropped a table by mistake at 9:01, then detected this error at 10.45, you are interested in getting the lost data as it was at 9:00 (just before the drop table). At the same time, you don't want to lose any new data added to other tables between 9:01 and 10:45. With databse cloning, you can create a clone of the database at 9:00 (before the table drop) and restore the lost data back to the production database.
```

Comment:
10.45 -> 10:45
databse -> database

Reply:
I also think we should be more clear about the restoration process.
"you can create a clone of the database at 9:00 (before the table drop) and restore the lost data back to the production database" -> "you can create a clone of the database as of 9:00 (before the table drop) and copy the data in the table from the cloned database to the original database"